### PR TITLE
solution

### PR DESCRIPTION
**Pull request by MargaritaChorniak**
[DEMO LINK](https://MargaritaChorniak.github.io/react_decompose/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,80 +1,16 @@
 import React from 'react';
 import './App.css';
+import Article from './components/Article/Article';
+import Header from './components/Header/Header';
+import Welcome from './components/Welcome/Welcome';
 // Move each BEM block to a separate component (file) and import them here
```
Comment (tetiana-ivanchenko) [inline src/App.jsx:6]: don't leave comments

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -4,3 +4,20 @@
 // Create a Header function returning the HTML of header block
 
 // Add a default export statement for Header component to use it in the other files
```
Comment (tetiana-ivanchenko) [inline src/components/Header/Header.jsx:6]: same for comments

File: src/components/Welcome/Welcome.jsx [hunk 3]
```diff
@@ -4,3 +4,15 @@
 // Create a Welcome function returning the HTML of welcome block
 
 // Add a default export statement for Welcome component to use it in the other files
```
Comment (tetiana-ivanchenko) [inline src/components/Welcome/Welcome.jsx:6]: same

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -1 +1,43 @@
...
+  opacity: 1;
+}
+
+@media (min-width: 500px) {
+  .header {
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+  }
+}
```
Comment (tetiana-ivanchenko) [inline src/components/Header/Header.css:43]: add media rules right after the classes you're applying them to

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/components/Article/Article.css [hunk 5]
```diff
@@ -1 +1,30 @@
...
+    margin: 3.5em auto 0;
+    padding: 2em;
+  }
+}
+
+@media (min-width: 800px) {
+  .article {
+    margin: 3.5em auto;
+  }
+}
```
Comment (tetiana-ivanchenko) [inline src/components/Article/Article.css:30]: same for thw media queries

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**